app-backend/migrations/src/main/scala/migrations/33.scala

```diff
@@ -0,0 +1,11 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M33 {
```

Comment:
Should make this a symlink like others. Should we make an issue to figure out why these even exist and if we can make this even simpler ?